src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
+import classNames from "classnames";
+
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useRoles } from "hooks/useRoles";
+import { useIsAdminUser } from "hooks/roles";
+
+import { Venue_v2 } from "types/venues";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueWizard from "pages/Admin/Venue/VenueWizard/VenueWizard";
+import { LoadingPage } from "components/molecules/LoadingPage";
+
+import "./AdminAdvancedSettings.scss";
+
+export enum AdminAdvancedTab {
+  basicInfo = "basic_info",
+  entranceExperience = "entrance_experience",
+  advancedMapSettings = "advanced_map_settings",
+}
+
+const adminAdvancedTabs: Readonly<Record<AdminAdvancedTab, String>> = {
+  [AdminAdvancedTab.basicInfo]: "Start",
+  [AdminAdvancedTab.entranceExperience]: "Entrance",
+  [AdminAdvancedTab.advancedMapSettings]: "Advanced",
+};
+
+const DEFAULT_TAB = AdminAdvancedTab.basicInfo;
+
+export const AdminAdvancedSettings: React.FC = () => {
+  const venueId = useVenueId();
+  const [selectedTab, setSelectedTab] = useState<string>(DEFAULT_TAB);
+
+  const { user } = useUser();
```

Comment:
userId

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1511/commits/484c4afb77a3b370f73be7ab7dbd7911b30553c0